pkg/cmd/create/create_devpod.go

```diff
@@ -158,6 +158,20 @@ func NewCmdCreateDevPod(commonOpts *opts.CommonOptions) *cobra.Command {
 	return cmd
 }
 
+// GetGitLabels generate label keys and values for git repository elements (scheme, host, org and repo)
+func GetGitLabels(gitInfo *gits.GitRepository) (string, string, string, string, string, string, string, string) {
```

Comment:
Why would you export this function? It should be at the very least private. With all these return parameters I find this function very hard to use. What is really the intention here? It is really easy to mess up the assignment in this way.